python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -266,6 +266,12 @@ def custom_repr_html(obj):
     html_formatter.for_type(DataFrame, custom_repr_html)
 
 
+def _Series_dtype(self):
+    # Fast-path to extract dtype from the current
+    # object without round-tripping through the slow<->fast
+    return self._fsproxy_wrapped.dtype
```

Comment:
I'm a little nervous to start doing this because we need to be sure the dtype between the fast and slow object are equal correct? e.g. If `_fsproxy_wrapped` is a `pandas.Series` with a the extension `pandas.Float64Dtype`, would that break anything?

Reply:
Wrapped the result with `_maybe_wrap_result`. Any custom extension types will not break because we will be failing in cudf anyways.